recipes/cobra/meta.yaml

```diff
@@ -0,0 +1,44 @@
+package:
+  name: cobra
+  version: "0.4.0b6"
+
+source:
+  git_url: https://github.com/opencobra/cobrapy.git
```

Comment:
I fixed a critical bug in the package and that has not been released yet. I have notified the authors for the same. Should make things easier.